.eslintrc

```diff
@@ -2,11 +2,23 @@
   "parser": "babel-eslint",
   "plugins": [
     "async-await",
-    "flowtype"
+    "flowtype",
+    "import"
   ],
   "settings": {
     "flowtype": {
       "onlyFilesWithFlowAnnotation": true
+    },
+    "import/ignore": [
+      // Because of CommonJS incompatibility, we can't
+      // check for bad imports in node_modules.
+      "node_modules", 
+    ],
+    "import/resolver": {
+      "node": {
+        // This adds ./src for relative imports.
+        "moduleDirectory": ["node_modules", "src"]
```

Comment:
Since our relative imports all use exact paths (unlike `addons-frontend`), adding `src` here isn't necessary. Also, since `node_modules` is a default, you can just remove this entire `import/resolver` block.